http/src/request/guild/member/update_guild_member.rs

```diff
@@ -12,13 +12,13 @@ use twilight_model::{
 #[derive(Clone, Debug)]
 pub enum UpdateGuildMemberError {
     /// The nickname is either empty or the length is more than 32 UTF-16 characters.
-    NicknameInvalid,
+    NicknameInvalid { nickname: String },
 }
 
 impl Display for UpdateGuildMemberError {
     fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
         match self {
-            Self::NicknameInvalid => f.write_str("the nickname length is invalid"),
+            Self::NicknameInvalid { .. } => f.write_str("the nickname length is invalid"),
```

Comment:
If we should include info in error formatting, should this include nickname length or the actual nickname? I'm thinking length because that's what is validated